server/db.js

```diff
@@ -0,0 +1,25 @@
+const mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost/reviews', { useNewUrlParser: true });
+
+
+const db = mongoose.connection;
+
+db.on('error', console.error.bind(console, 'connection error:'));
+db.once('open', () => console.log('connected to DB'));
+
+let reviewSchema = new mongoose.Schema({
+  author: String,
+  rating: Number,
+  date: String,
+  review: {
+    tittle: String,
```

Comment:
Is it typo @ line 15 'tittle'?

Reply:
Nice catch. Thanks, this has been adjusted.